src/llmq/params.h

```diff
@@ -265,7 +265,7 @@ static constexpr std::array<LLMQParams, 10> available_llmqs = {
         .dkgBadVotesThreshold = 48,
 
         .signingActiveQuorumCount = 32,
-        .keepOldConnections = 33,
+        .keepOldConnections = 64,
```

Comment:
should do the same for `LLMQ_TEST_DIP0024`

Reply:
@ogabrielides I'm a little confused how you decided on the value `.keepOldConnections = 64,`, so I pretty arbitrarily chose 5 for LLMQ_TEST_DIP0024. Please explain how you decided this value.

Reply:
@PastaPastaPasta In non-rotation, 1 new quorum is created in each cycle.
So that’s why `keepOldConnections` should be at least one more than the active quorums set.
Since with rotation, `signingActiveQuorumCount` quorums are created in each cycle, we need to adjust `keepOldConnections` to `2 x signingActiveQuorumCount`